modules/codegen/src/main/scala/dev/guardrail/generators/Java/JacksonGenerator.scala

```diff
@@ -126,7 +126,6 @@ object JacksonGenerator {
                     new AssignExpr(
                       new FieldAccessExpr(new ThisExpr, term.parameterName),
                       term.fieldType match {
-                        case _: PrimitiveType => new NameExpr(term.parameterName)
```

Comment:
The idea is to remove this because this case will never happen anymore; we _always_ want to wrap these in `requireNonNull()` even if the underlying (field) type is a primitive, the constructor param will always be boxed.  In reality, though, we don't actually need to use `requireNonNull()` here, because attempting to assign a `null` boxed value to its corresponding primitive type will throw an NPE.  But `requireNonNull()` is useful to show intent, and to get a better error message.